src/main/cpp/fielddata/FieldElementCalculator.cpp

```diff
@@ -81,50 +81,50 @@ void FieldElementCalculator::InitializeReefBranchTransformsMap()
     m_reefBranchOffsetMap[OffsetEnums::COMP_LEFT_BLUE] = frc::Transform3d(
         frc::Translation3d(
             units::length::inch_t(0.0),
-            units::length::inch_t(-7.5),
+            units::length::inch_t(-6.5),
```

Comment:
We should create a story to define these as constants in the .h file (for instance, the -6.5 inches could be re-used multiple times limiting our potential errors).